data/obs-packaging/cc-oci-runtime/README.md

```diff
@@ -0,0 +1,18 @@
+#cc-oci-runtime
+
+This directory contains the sources to update rpm specfiles and debian source
+control files to generate ``cc-oci-runtime`` packages for Fedora and Ubuntu.
+
+There is a script to generate specs and dsc automatically, to be pushed to
+Open Build Service (http://openbuildservice.org/) and generate packages for you.
+
+
+``./update_runtime.sh``
+
+This script will generate/update the sources to create ``cc-oci-runtime`` packages.
```

Comment:
This script generates and updates the sources to create the ``cc-oci-runtime`` packages.